src/auth.ml

```diff
@@ -230,22 +233,20 @@ module External_account_credentials = struct
     |> to_string
 
   let subject_token_of_response (t : t)
-      ((resp, body) : Cohttp.Response.t * Cohttp_lwt.Body.t) :
+      ((resp, body_str) : Cohttp.Response.t * string) :
```

Comment:
Helper functions now take a `string` instead of the `Cohttp_lwt.Body.t`, to encourange consuming the body at the request call site.